RELEASE_NOTES.md

```diff
@@ -1,76 +1,53 @@
-# ChefDK 0.10.0 Release notes
+# ChefDK 0.11 Release notes
 
 ## New / Updated Packages
 
-* inspec - new dependency, version 0.9.0
-* kitchen-inspec - new dependency, version 0.9.0
-* knife-windows - updated to 1.1.1
-* chef-provisioning - updated to 1.5.0
-* chef-provisioning-aws - updated to 1.6.1
+This version of ChefDK includes a number of key updates to dependent packages:
 
-## Inspec
+* chef - updated to 12.7.2
```

Comment:
I think this should link to the Chef 12.7.2 release notes.  Whether we do the same for everything else... Seems like we could do it for Ohai, Inspec, Kitchen-inspec and test-kitchen (because I know those all had at least minor version bumps)